src/js/Core.js

```diff
@@ -192,6 +192,7 @@ Wui.percentToPixels = function(el,percent,dim){
 
 
 Wui.positionItem = function(parent,child){
+    var win = $(parent).closest('.w121-window');
```

Comment:
I'm interested in why this choice was made. The code appears to reduce the use of positionElement to identifying fixed position elements only within Wui.Window's instead of inside any fixed position element. I'm worried that this would also conflict in cases where an object is not in a Wui.Window, but in a fixed position element, and then gets the scrollTop added (which would likely render the item invisible).